test/org/sagebionetworks/bridge/services/ParticipantServiceTest.java

```diff
@@ -938,21 +933,6 @@ public void throwLimitExceededException() {
         participantService.createParticipant(STUDY, CALLER_ROLES, PARTICIPANT, false);
     }
     
-    private void verifyStatusCreate(Set<Roles> callerRoles) {
-        mockHealthCodeAndAccountRetrieval();
-        
-        StudyParticipant participant = new StudyParticipant.Builder().copyOf(PARTICIPANT)
-                .withStatus(AccountStatus.ENABLED).build();
```

Comment:
Why were these tests deleted?

Reply:
These tests were supposed to verify that status was not changed, but it is now changed in the service (and that is now verified in other tests, like those that call verifyStatusUpdate... which looks at impact of roles on whether or not you can change the status, and the tests you just asked me to add to test calls to accountWorkflowService).